skimage/transform/_geometric.py

```diff
@@ -1197,11 +1197,12 @@ class EuclideanTransform(ProjectiveTransform):
     translation parameters. The similarity transformation extends the Euclidean
     transformation with a single scaling factor.
 
-    In 2D and 3D, the transformation parameters can be given as the homogeneous
-    transformation matrix, above, or as the implicit parameters, rotation and
-    translation in x (a1) and y (b1). Beyond 3D, translation can be specified
-    using the implicit parameter (again, in x (a1), y (b1), etc.), otherwise,
-    only the matrix form is allowed.
+    In 2D and 3D, the transformation parameters may be provided either via
+    `matrix`, the homogeneous transformation matrix, above, or through the
```

Comment:
```suggestion
    `matrix`, the homogeneous transformation matrix, above, or via the
```
Just so that the EITHER/OR instruction is (perfectly parallel and, hence,) crystal clear to the reader. I've noticed that I used 'passed through' in my https://github.com/scikit-image/scikit-image/pull/7103#discussion_r1330595815 but since you went for 'provided via,' we might as well stick with 'via.'